media-capabilities/decodingInfo-webrtc.any.js

```diff
@@ -225,3 +372,79 @@ validVideoCodecs.forEach(codec => {
 }, "Test that decodingInfo returns supported true for the codec " + codec + (isWorkerEnvironment ? "" : " returned by RTCRtpReceiver.getCapabilities()"))}
 );
 
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'webrtc',
+    audio: { contentType: 'audio/webm; codecs="opus"; foo="bar"' },
+  }));
+}, "Test that decodingInfo rejects if the audio configuration contentType has more than one parameters");
+
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'webrtc',
+    audio: { contentType: 'audio/webm; foo="bar"' },
+  }));
+      }, "Test that decodingInfo rejects if the audio configuration contentType has one parameter that isn't codecs");
+
+promise_test(t => {
```

Comment:
That test case seems to be a copy and paste from `decodingInfo.any.js` applied to the `webrtc` case but, if I read the spec correctly, `hdrMetadataType`, `colorGamut`, or `transferFunction` are not applicable to the `webrtc` case, and thus create an invalid MediaConfiguration.

In other words, I would expect the call to `decodingInfo()` to return a rejected promise when type is `webrtc`.

Reply:
Reworked this test to check that `decodingInfo` rejects for this case.